openforcefield/tests/test_wbo.py

```diff
@@ -0,0 +1,46 @@
+# will merge this file with test_toolkits.py later, currently doing it separately
+
+
+from simtk.openmm.app import PDBFile
+from openforcefield.topology import Molecule, Topology
+from openforcefield.typing.engines.smirnoff import ForceField
+from openforcefield.utils import get_data_file_path
+
+
+## Create a simple molecule from SMILES and turn it into a topology.
+#mol1 = Molecule.from_smiles("C1=NC=CC(=C1)C2=CC=C(C=C2)O")
+#mol1.generate_conformers(n_conformers=1)
+#mol1.compute_wiberg_bond_orders()
+#
+#mol2 = Molecule.from_smiles("C1=NC=CC(=C1)C2=CC=C(C=C2)[O-]")
+#mol2.generate_conformers(n_conformers=1)
+#mol2.compute_wiberg_bond_orders()
+
+# Reading neutral molecule from file
+filename = get_data_file_path('molecules/molecule_neutral.sdf')
```

Comment:
Thanks for using `get_data_file_path` -- It's good that our tests use the same infrastructure.